src/service/viewer-interface.js

```diff
@@ -162,6 +162,11 @@ export class ViewerInterface {
    */
```

Comment:
(Sorry GH won't let me comment on line 160) Does the type need to updated to `nullable`?

Reply:
Not the setter